src/components/common/toast/Toast.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+import { useDispatch } from 'react-redux';
+import { Alert, Snackbar } from '@mui/material';
+
+import useAppSelector from '@/hooks/useAppSelector';
+import { hideToast } from '@/redux/reducers/toast.reducer';
+import styles from './Toast.module.scss';
+const Toast: FC = () => {
```

Comment:
I'd add space between `import styles from './Toast.module.scss` and `Toast`

Reply:
fixed